web/src/pages/Channel/EditChannel.js

```
@@ -964,17 +1034,38 @@
       }
       setInitialKey(data.key); // Store initial key for single input mode placeholder
 
+      let processedModelMapping = '';
```

Biome (2.1.2)

[error] setting is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)